src/BaseballGame.js

```diff
@@ -0,0 +1,74 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { generateRandomNum } from "./utils/generateRandomNum";
+import { isInputValid, isEndInputValid } from "./utils/validation";
+import { getResult, printResult } from "./utils/getResult";
+
+class BaseballGame {
+  #COMPUTER_NUM_LEN = 3;
+  #ComputerNums;
+  #UserNums;
+
+  setComputerNums(nums) {
+    this.#ComputerNums = nums;
+  }
+
+  setUserNums(nums) {
+    isInputValid(nums, this.#COMPUTER_NUM_LEN);
+    this.#UserNums = nums.split("").map((str) => Number(str));
```

Comment:
```suggestion
    this.#UserNums = nums.split("").map(Number);
```
사소하지만 요렇게 축약해도 동작은 같을 것 같아요 ㅇ_ㅇ!

Reply:
그러게요.. 찾아보니 그렇게도 가능하네요 하핳 왜 어려운 길을 가고 있었을까.. ㅋㅋㅋ
감사합니다!

Reply:
그래도 지예님 코드가 더 목적성있고 의도가 보이는 것 같기도 하네요!